### ISSUE TITLE
[feature request] Add option for whitelist(positive search)

### ISSUE THREAD

**Issue by ResistanceIsUseless** [bp0lr/gauplus#6]
it would be nice to specify only what file types you want like -w php,txt,xhtml 

thanks!

**Comment by bp0lr** [comment]
Hello!,

Would be great, but I don't have the time to make it happen.
But you can do something like 
```grep -vE "robot.txt$|robots.txt\/$|.gif|.jpg|.jpeg|.css|.tiff|.ico|.png|.svg|.ttf|.woff|.woff2|.eot|.pdf|.js" "gauplus_results.txt" > "gauplus_clean.txt"```

change it to whatever you need.

**Comment by ResistanceIsUseless** [comment]
True, piping through grep was my backup. I could also fork the repo and try and make the changes myself. Anyways, i figured i would ask.

Thanks for a great tool and a quick response!
